### PR TITLE
Update index: fig, theory

### PR DESCRIPTION
**Pull request by jabascal**
Updated index

### CODE REVIEW

**Comment by nducros** [comment]
**Intro**

SPyRiT is a [PyTorch](https://pytorch.org/)-based package for deep image reconstruction that is primarily designed for single-pixel imaging. SPyRiT can simulate measurements and perform image reconstruction using several data-driven reconstruction algorithms.

As shown below, SPyRiT defines full networks that comprise a measurement operator, a noise operator, a preprocessing operator, a reconstruction operator, and a learnable neural network. All operators inherit from PyTorch nn.Module class and easily combine into a full network.

**Comment by nducros** [comment]
Package description -> Single-pixel imaging
Theory -> Modelling of the measurements
Add "Image reconstruction" at the same level as "Modelling of the measurements"

The reconstruction operator -> We consider the typical reconstruction operator XX that can be written

is a learned neural network -> is a neural network that can be learnt during the training phase. Alternatively, XX can be simply "plugged". In this case, its training is performed beforehand.

a forward pass can be written as follows

The full network is trained by minimizing the following loss function: - > The full network can be trained using a database that contains images only: 

Its main functionalities - > The main functionalities os SPyRiT

**Comment by nducros** [comment]
Can you remind me here the diagram is stored?
